frontend/src/scenes/notebooks/Nodes/NotebookNodeFlagCodeExample.tsx

```diff
@@ -0,0 +1,47 @@
+import { createPostHogWidgetNode } from 'scenes/notebooks/Nodes/NodeWrapper'
+import { NotebookNodeType } from '~/types'
+import { useValues } from 'kea'
+import { FeatureFlagLogicProps, featureFlagLogic } from 'scenes/feature-flags/featureFlagLogic'
+import { FeatureFlagCodeExample } from 'scenes/feature-flags/FeatureFlagCodeExample'
+import { urls } from 'scenes/urls'
+import { JSONContent, NotebookNodeViewProps } from '../Notebook/utils'
+import { notebookNodeLogic } from './notebookNodeLogic'
+
+const Component = (props: NotebookNodeViewProps<NotebookNodeFlagCodeExampleAttributes>): JSX.Element => {
+    const { id } = props.node.attrs
+    const { featureFlag } = useValues(featureFlagLogic({ id }))
+    const { expanded } = useValues(notebookNodeLogic)
+
+    return <div className="p-2">{expanded && <FeatureFlagCodeExample featureFlag={featureFlag} />}</div>
+}
+
+type NotebookNodeFlagCodeExampleAttributes = {
+    id: FeatureFlagLogicProps['id']
+    expandedOnLoad?: boolean
+}
+
+export const NotebookNodeFlagCodeExample = createPostHogWidgetNode<NotebookNodeFlagCodeExampleAttributes>({
+    nodeType: NotebookNodeType.FeatureFlagCodeExample,
+    title: 'Feature Flag Code Example',
+    Component,
+    heightEstimate: '3rem',
+    href: (attrs) => urls.featureFlag(attrs.id),
+    resizeable: false,
+    attributes: {
+        id: {},
+        expandedOnLoad: {},
```

Comment:
by the way, what do these attributes do? Noticed typing wasn't happy when I added this optional prop, needed to add this here

Reply:
Attributes are specific to individual nodes and can be edited for specific nodes. I think what you want here is the `startExpanded` prop rather than an attribute